LibertyDSNP/schemas
# Create npm Schemas Library for easy schema and parquet creation access

Problem
=======
As a developer, I want to be able to easily interact with the schemas and create parquet files from the schemas

Solution
========
Lots of stuff, but in general getting it so it can be an npm package.

Change summary:
---------------
* Upgraded to the latest polkadotjs
* Deployment is now always in the same order
* Moved the deploy and read scripts into a cli directory
* Added package build and release logic
* Added a parquet helper that can build the correct parquet files `fromFrequencySchema`

Steps to Verify (Library):
----------------
1. Clone
1. npm i
1. npm run build
1. cd dist && npm pack
1. Create a different npm package and follow the instructions on the readme using the packed tgz file.

Steps to Verify (Deploy):
----------------
1. Run Frequency in instant sealing mode
2. npm run deploy
3. npm run read
